tests/core/test_agent.py

```diff
@@ -237,13 +236,12 @@ def test_form_without_form_policy(policy_config: Dict[Text, List[Text]]):
             domain=Domain.from_dict({"forms": {"restaurant_form": {}}}),
             policies=PolicyEnsemble.from_dict(policy_config),
         )
-    assert "neither added the 'RulePolicy' nor the 'FormPolicy'" in str(execinfo.value)
+    assert "have not added the 'RulePolicy'" in str(execinfo.value)
 
 
 @pytest.mark.parametrize(
     "policy_config",
     [
-        {"policies": [{"name": FormPolicy.__name__}]},
         {"policies": [{"name": RulePolicy.__name__}]},
```

Comment:
I guess we don't need to `parametrize` now?

Reply:
Removing parametrization.